scripts/get_summary.py

```diff
@@ -76,29 +60,32 @@ def parse_environment_variables():
     output_date = os.getenv("OUTPUT_DATE")
 
     return {
-        'output_dir': output_dir,
-        'number_of_days': days,
-        'output_date': output_date,
+        "output_dir": output_dir,
+        "number_of_days": days,
+        "output_date": output_date,
     }
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     parsed_variables = parse_environment_variables()
-    date_to_output = parsed_variables['output_date']
+    date_to_output = parsed_variables["output_date"]
 
     sorted_folders = sorted(glob(f"{parsed_variables['output_dir']}/*"))
     result = get_results(sorted_folders)
 
     for result in result[::-1]:
         summary = (
-            f'Questionnaire GETs average: {int(result[1])}ms\n'
-            f'Questionnaire POSTs average: {int(result[2])}ms\n'
-            f'All requests average: {int(result[3])}ms'
+            f"Questionnaire GETs average: {int(result[1])}ms\n"
+            f"Questionnaire POSTs average: {int(result[2])}ms\n"
+            f"All requests average: {int(result[3])}ms\n"
+            f"Total Requests: {int(result[4])}\n"
+            f"Total Failures: {int(result[5])}\n"
+            f"Error Percentage: {(round(result[6], 2))}%"
         )
```

Comment:
My comment about `__str__` applies here also.

Reply:
Refactored this file so it uses the new class